crypto/verifier.go

```diff
@@ -0,0 +1,6 @@
+package crypto
+
+// Verifier defines an interface to verify if the message and the signature are matched.
+type Varifier interface {
```

Comment:
```suggestion
type Verifier interface {
```

Reply:
@thongdk8 
Thank you.
Let me fix this.

Reply:
Fixes in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/6/commits/87e692b4ea5483c7a77067d7e50ed199632f0d1c